src/app/Program.cs

```diff
@@ -238,66 +244,5 @@ private static async Task<IWebHost> BuildHost(string kvUrl, AuthenticationType a
             // build the host
             return builder.Build();
         }
-
```

Comment:
Is there a reason you deleted this? It's still needed for "proactive" key rotation vs. "reactive"

Reply:
I have added it back to the code. My thought was that it was being refactored within its own class. If we have to remove the code again, we can do so later.